### ISSUE TITLE
Job Results - Details

### ISSUE THREAD

**Issue by mabashian** [ansible/awx#4223]
## Job Results - Details

Mockup: https://tower-mockups.testing.ansible.com/patternfly/jobs/jobs-detail/

Depends on #219 

The data for the details comes from `/api/v2/jobs/:id`.  If the header work has already been done then this request will have already been made when the page is loaded.  This issue assumes that displaying extra variables has already been established.  If not, then it is not in the scope of this issue.

#### Tasks
* Display all the fields including additional details section

_Job Details Mockup_ 
<img width="1317" alt="Screen Shot 2019-05-29 at 1 42 16 PM" src="https://user-images.githubusercontent.com/15881645/58578796-a116de80-8217-11e9-8b55-c6f94417a134.png">

#### Not in Scope for MVP
* Copy revision icon/logic
* Workflow icon shown next to template name if job is part of a workflow
* Delete - see #4222

**Comment by mabashian** [comment]
Marked https://github.com/ansible/awx/issues/4222 as blocked until this lands - when it does we can remove the label from that issue.
